### PR TITLE
[Feature] Add voxelpose demo

### PR DESCRIPTION
**Pull request by wusize**
## Motivation
Add demo/inference for voxelpose

## Checklist

**Before PR**:

- [ ] I have read and followed the workflow indicated in the [CONTRIBUTING.md](https://github.com/open-mmlab/mmpose/blob/master/.github/CONTRIBUTING.md) to create this PR.
- [ ] Pre-commit or linting tools indicated in [CONTRIBUTING.md](https://github.com/open-mmlab/mmpose/blob/master/.github/CONTRIBUTING.md) are used to fix the potential lint issues.
- [ ] Bug fixes are covered by unit tests, the case that causes the bug should be added in the unit tests.
- [ ] New functionalities are covered by complete unit tests. If not, please add more unit tests to ensure correctness.
- [ ] The documentation has been modified accordingly, including docstring or example tutorials.

**After PR**:

- [ ] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: mmpose/core/visualization/image.py [hunk 1]
```diff
@@ -440,3 +440,75 @@ def imshow_mesh_3d(img,
             valid_mask * color[:, :, :3] + (1 - valid_mask) * output_img)
 
     return output_img
+
+
+def imshow_multiview_keypoints_3d(
```
Comment (ly015) [inline mmpose/core/visualization/image.py:445]: Could you add a unittest for this function?

**Comment by ly015** [comment]
The image data size is still a little bit large. How about we package the images as a `.zip` or `.tar` file and hold it on OSS. We can guide the user to download the data in the markdown doc, or automatically download it in the script.

**Comment by jin-s13** [comment]
unittest coverage rate should be further improved.

**Comment by JonathanLehner** [comment]
how is it going? can the demo already be used?

**Comment by jin-s13** [comment]
> how is it going? can the demo already be used?

Thanks for your interest! @JonathanLehner 

It is still in review, and some modifications are required.
But I think basic functions have been realized. You may have a try!

Also it is very welcome if you could help review this PR. If you have any comments about this PR, please feel free to comment here directly.

**Review by jin-s13 — APPROVED** [review]

**Comment by JonathanLehner** [comment]
Thanks everyone for your great work! It seems you finished now? I will have more time tomorrow, then I can debug

**Comment by ly015** [comment]
> Thanks everyone for your great work! It seems you finished now? I will have more time tomorrow, then I can debug

Thank you very much. This PR has been tested and merged. Welcome to have a try and please leave us an issue if you run into any problems or have any suggestions. We are looking forward to your feedback.

**Review by JonathanLehner — COMMENTED** [review]
looks all good. outside of this PR, I also get this warning: __floordiv__ is deprecated, and its behavior will change in a future version of pytorch. It currently rounds toward 0 (like the 'trunc' function NOT 'floor'). This results in incorrect rounding for negative values. To keep the current behavior, use torch.div(a, b, rounding_mode='trunc'), or for actual floor division, use torch.div(a, b, rounding_mode='floor'). -- that might be good to replace

**Comment by wusize** [comment]
> 

Thank you Jonathan! This warning is raised for operation "a // b" with pytorch version beyond 1.9.0. I will replace it.

**Pull request merged**